src/iotop.h

```diff
@@ -70,10 +70,19 @@ typedef union {
 		int sort_order;
 		int base; // 1000 or 1024
 		int threshold; // 1..10
+		int charset;
 	} f;
 	int opts[22];
 } config_t;
 
+
+// character sets for pseudographics
+enum {
+	CHARSET_BRAILLE,
+	CHARSET_ASCII,
+	CHARSET_97531
```

Comment:
97531 is a terrible name.

Reply:
Odd numbers, reversed ;)

Maybe it is OK to use internally but what is exposed to users should be more human readable